### PR TITLE
Python 3 support

### LINKED ISSUE #434: Errors running Make with Python 3
```
What steps will reproduce the problem?
1. Install Python 3.
2. Attempt to run 'make', which throws syntax errors in ./scripts/*.py.

What is the expected output? What do you see instead?
Expected it to use compile cleanly using Python 2, not the newer system default.

What version of Google Test are you using? On what operating system?
gtest-1.6.0, Arch Linux
```

Original issue reported on code.google.com by `egt...@gmail.com` on 19 May 2013 at 5:49

### PR DESCRIPTION
**Pull request by tamland**
Adds support for running googletests own tests and the 'fuse' script with python 3. Fixes #434

### CODE REVIEW

File: googletest/scripts/fuse_gtest_files.py [hunk 1]
```diff
@@ -90,10 +93,10 @@ def VerifyFileExists(directory, relative_path):
   """
 
   if not os.path.isfile(os.path.join(directory, relative_path)):
-    print 'ERROR: Cannot find %s in directory %s.' % (relative_path,
-                                                      directory)
-    print ('Please either specify a valid project root directory '
-           'or omit it on the command line.')
+    print('ERROR: Cannot find %s in directory %s.' % (relative_path,
```
Comment (BillyDonahue) [inline googletest/scripts/fuse_gtest_files.py]: I'm not a python expert, but doesn't this print function syntax require an import from the future?

Comment (tamland) [inline googletest/scripts/fuse_gtest_files.py]: Nope, that's only for *enforcing* new style syntax. Haven't done that as I've tried to keep python2.3 support (the print_function future is only available in 2.6+) ;)

**Comment by tamland** [comment]
@BillyDonahue Will this be accepted?

**Comment by BillyDonahue** [comment]
So the parens syntax is ok in 2.3?

**Comment by tamland** [comment]
Yes, it will work in python 2.3. The parentheses is a widely used 'hack' for writing python 2/3 compatible code (in 2.x it will be interpreted the same, a keyword + an expression, only enclosed in parentheses. Nothing special)

**Pull request merged**